x/slashing/migrations/v4/keys.go

```diff
@@ -11,8 +11,13 @@ import (
 const MissedBlockBitmapChunkSize = 1024 // 2^10 bits
 
 var (
-	ValidatorSigningInfoKeyPrefix         = []byte{0x01}
-	validatorMissedBlockBitArrayKeyPrefix = []byte{0x02}
+	ValidatorSigningInfoKeyPrefix                   = []byte{0x01}
+	deprecatedValidatorMissedBlockBitArrayKeyPrefix = []byte{0x02}
+
+	// NOTE: sdk v0.50 uses the same key prefix for both deprecated and new missed block bitmaps.
+	// We needed to use a new key, because we are skipping deletion of all old keys at upgrade time
+	// due to how long this would bring the chain down. We use 0x10 here to prevent overlap with any future keys.
+	validatorMissedBlockBitMapKeyPrefix = []byte{0x10}
```

Comment:
Set to 0x10 to prevent overlap if more keys are introduced in the next sdk release we upgrade to